examples/keygen/keygen.c

```diff
@@ -55,12 +106,15 @@ int TPM2_Keygen_Example(void* userCtx, int argc, char *argv[])
     WOLFTPM2_KEYBLOB newKey;
     TPMT_PUBLIC publicTemplate;
     TPMI_ALG_PUBLIC alg = TPM_ALG_RSA; /* default, see usage() for options */
+    TPM_ALG_ID algSym = TPM_ALG_CTR; /* default Symmetric Cypher, see usage */
     TPM_ALG_ID paramEncAlg = TPM_ALG_NULL;
     WOLFTPM2_SESSION tpmSession;
     TPM2B_AUTH auth;
     int bAIK = 1;
     int keyBits = 256;
     const char* outputFile = "keyblob.bin";
+    unsigned long len = 0;
+    char symMode[7] = "aesctr";
```

Comment:
The `[7]` can be `[]` and then it will include the null terminator.

Reply:
Indeed.